pkg/apis/policy/v1alpha1/clusterimagepolicy_defaults.go

```diff
@@ -29,5 +31,8 @@ func (spec *ClusterImagePolicySpec) SetDefaults(ctx context.Context) {
 		if authority.Name == "" {
 			spec.Authorities[i].Name = fmt.Sprintf("authority-%d", i)
 		}
+		if authority.Key == nil && authority.Static == nil && authority.Keyless != nil && authority.Keyless.CACert == nil && authority.Keyless.URL == nil {
+			authority.Keyless.URL = apis.HTTPS("fulcio.sigstore.dev")
+		}
```

Comment:
does the ctlog url need to be set as well? we probably need to point to `rekor.sigstore.dev`